### DISCUSSION TITLE
Episode 159: The Roc Programming Language with Richard Feldman (Part 3)

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#51] [Episode]
Episode Link: https://adspthepodcast.com/2023/12/08/Episode-159.html

Feel free to write comments or ask questions 🙂

**Comment by BraxtonMDevelops** [comment]
Hi to Conor and Bryce I'm curious as to if you all in coming weeks will give a schedule for like conferences you're going to in 2024

**Reply by codereport** [reply]
Yep, usually in the first couple weeks of each new year we do a previous year retro and a upcoming year preview. So look out for those.

**Reply by BraxtonMDevelops** [reply]
alright great.  Just got done with the backlog of ADSP, and i started listening proper around 150 so I don't know. I would pay some amount of money for what I dub "the Squirrel Cut" where we get the Bryce tangents that like don't allow listeners to dox him and such for a given amount of money. Much love to the cast and am beginnning my journey through Array Cast now.

**Comment by flandweber** [comment]
Just as a note to Richards explanation of the dependency specification with checksums: Nix does it exactly the same way.
https://ryantm.github.io/nixpkgs/builders/fetchers
